eogrow/utils/eopatch_list.py

```diff
@@ -11,16 +11,16 @@
 from ..types import PatchList
 
 
-def save_eopatch_names(filesystem: FS, file_path: str, eopatch_list: List[str]) -> None:
+def save_eopatch_names(filesystem: FS, file_path: str, eopatch_names: List[str]) -> None:
```

Comment:
should we use consistency also here and use `patch_names`/`patch_names_list`?

Reply:
not sure, technically this is more general. If anything, it should be changed to 'save_execution_names` and `load_eopatch_names` but that is just confusing :see_no_evil: 

Perhaps just `save_names` and `load_names`?

Reply:
Sorry, I was talking about the argument `eopatch_names` here.

But your comment is relevant, so if it should be more general, then I like your last suggestions.

Reply:
changed to `save_names` and `load_names`